.github/actions/get-version-number-and-issue-url-v1/README.md

```diff
@@ -0,0 +1,43 @@
+# get-version-number-and-issue-url-v1
+
+This action gets the version number and issue url from the release-candidate.
+
+## Inputs
+
+| Input                   | Description                                                   | Required | Default |
+| ----------------------- | ------------------------------------------------------------- | -------- | ------- |
+| `disable-get-issue-url` | Whether or not to get the issue URL for the release candidate | false    | `false` |
```

Comment:
If you are providing an input to disable half of this action, it would probably be best to split this action out into two separate actions